x-pack/plugins/security_solution/public/entity_analytics/components/entity_details_flyout/tabs/risk_inputs.test.tsx

```diff
@@ -58,7 +58,7 @@ describe('RiskInputsTab', () => {
       </TestProviders>
     );
 
-    expect(getByTestId('risk-input-tab-title')).toBeInTheDocument();
+    expect(getByTestId('risk-input-contexts-title')).toBeInTheDocument();
     expect(getByTestId('risk-input-table-description-cell')).toHaveTextContent(
       'Risk inputRule Name'
     );
```

Comment:
Could you add some basic test scenarios for the `ContextsTable` component?

Reply:
Done in commit [unit test contexts table](https://github.com/elastic/kibana/pull/174638/commits/f2dd6ba6e4d1d781d7a4f7c4d109634b2e3dc6c4)